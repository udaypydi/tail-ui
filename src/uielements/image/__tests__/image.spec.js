import React from 'react';
import { render } from 'test-utils';
import  Image from 'uielements/image/image.component.js';

it('should take a snapshot', () => {
    const { asFragment } = render(<Image src="https://www.google.com" />)
    
    expect(asFragment(<Image src="https://www.google.com" />)).toMatchSnapshot()
});
