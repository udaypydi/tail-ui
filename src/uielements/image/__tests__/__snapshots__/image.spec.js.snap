// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should take a snapshot 1`] = `
<DocumentFragment>
  <img
    alt=""
    class=""
    role="presentation"
    src="https://www.google.com"
  />
</DocumentFragment>
`;
